controller/controller.js

```diff
@@ -5,3 +5,16 @@ exports.getAllTopics = (req, res) => {
     res.status(200).send(response);
   });
 };
+
+exports.deleteCommentById = (req, res, next) => {
+  const articleId = req.params.comment_id;
+
+  model
+    .deleteComment(articleId)
+    .then(() => {
+      res.status(204).send({});
```

Comment:
might be cleaner to do `res.sendStatus(204)` - then dont need the empty send bit